crate/multiworld-gui/src/main.rs

```diff
@@ -1514,6 +1525,7 @@ impl Application for State {
                 #[cfg(not(any(target_os = "linux", target_os = "windows")))] Frontend::BizHawk => unreachable!("no BizHawk support on this platform"),
                 Frontend::Pj64V3 => {
                     col = col
+                        .push(Button::new("Open Project64").on_press(Message::LaunchProject64))
```

Comment:
I think it makes more sense to move this button below the “waiting for Project64…” row, since it's essentially a step of the instructions. It might even make sense to turn it into a `Row` with a “1.” before it, and renumber the remaining steps. In the case where the config entry is missing as discussed above, the button could be plain text instead.

Reply:
I updated the ordering of elements and added a step for opening Pj64. 
i aligned the elements in the row at center, but i can align it bottom if you think it would look better

Reply:
Center alignment looks good imo. Probably something worth reviewing for other rows with a mix of text and buttons as well.